core/tests/karma-globals.js

```diff
@@ -69,6 +69,12 @@ var GLOBALS = {
       },
       is_trainable: true,
       narrow_instructions: null
+    },
+    Continue: {
```

Comment:
I'm not sure about this / am a bit confused. Should I be pulling these from here or elsewhere?

Reply:
Take a look at exploration_editor/editor_tab/SolutionVerificationServiceSpec.js -- I would suggest $provide-ing it as a constant in the relevant test spec instead.

Reply:
Done